org.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
+- --share=ipc
+- --socket=x11
+- --socket=wayland
```

Comment:
does it support wayland? If yes, then it's "fallback-x11". If not then the wayland socket is removed.

Reply:
i tested it and it runs just fine with `fallback-x11` and `x11`

Reply:
removed wayland